frontend/ui/components/_general/Input/Input.jsx

```diff
@@ -1,33 +1,45 @@
-import css from './Input.module.css';
+import cn from './Input.module.scss';
+import { BackspaceOutline } from '../icons/BackspaceOutline.jsx';
+import { CopyButton } from '../CopyButton/CopyButton.jsx';
+import { useWatch } from 'react-hook-form';
 
 export const Input = ({
   register = () => ({}),
+  control,
   id,
   name,
   placeholder,
-  textarea = false,
-  cols,
-  rows,
+  label,
+  copy,
+  clear,
+  error = false,
   type = 'text',
   disabled = false,
-}) =>
-  textarea ? (
-    <textarea
-      {...register(name)}
-      id={id || name}
-      placeholder={placeholder}
-      cols={cols}
-      rows={rows}
-      className={`${css.input} ${css.textarea}`}
-      disabled={disabled}
-    />
-  ) : (
-    <input
-      {...register(name)}
-      id={id || name}
-      placeholder={placeholder}
-      className={css.input}
-      type={type}
-      disabled={disabled}
-    />
+}) => {
+  const value = useWatch({
```

Comment:
Why do we need useWatch here?

Reply:
We need this for hiding buttons when input is empty